### DISCUSSION TITLE
Renovate Automerge for changed Branches

### DISCUSSION THREAD

**Discussion by MartinWitt** [renovatebot/renovate#21798] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Renovate merges the branch even when a commit from github actions was commited to the branch

### What you tried so far.

https://github.com/chains-project/maven-lockfile/pull/175 This is the current state. I have a github action which modifies every pull request to update the lockfile. I would like to get this automerged or even a config to disable the warning for github actions. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Use gitIgnoredAuthors
